hbw/app/assets/javascripts/hbw/forms.js.coffee

```diff
@@ -3,8 +3,9 @@ modulejs.define 'HBWForms', ['jQuery'], (jQuery) ->
     whenFinished: null
     onStart: ->
     onFinish: ->
+      @dispatcher.trigger('hbw:form_loaded')
```

Comment:
It's quite oviously a bad decision because it requires you to introduce a dispatcher dependency just for that.
Forms is a low-level transport not intended for trigger event for you. Also AFAIK event names should be hyphened/dashed, not underscored